batch/executor_test.go

```diff
@@ -16,7 +16,7 @@ import (
 var ErrTest = errors.New("unit test error")
 
 func TestBatch(t *testing.T) {
-	require := require.New(t)
+	req := require.New(t)
```

Comment:
This change was for a lint warning since there's also a "require" package being imported.